demisto_sdk/commands/common/hook_validations/structure.py

```diff
@@ -114,8 +114,9 @@ def is_valid_scheme(self):
             # disabling massages of level INFO and beneath of pykwalify such as: INFO:pykwalify.core:validation.valid
             log = logging.getLogger('pykwalify.core')
             log.setLevel(logging.WARNING)
+            scheme_file_name = 'integration' if self.scheme_name == 'betaintegration' else self.scheme_name
```

Comment:
can't we point betaintegation to integration scheme ?

Reply:
this is exactly what is done here. we are pointing the beta to the integration scheme.